addons/html_editor/static/src/main/font/color_plugin.js

```diff
@@ -105,12 +108,25 @@ export class ColorPlugin extends Plugin {
         const hasGradient = isColorGradient(backgroundImage);
         const hasTextGradientClass = el.classList.contains("text-gradient");
 
+        let backgroundColor = elStyle.backgroundColor;
+        const activeTab = document
+            .querySelector(".o_font_color_selector button.active")
+            ?.innerHTML.trim();
+        if (backgroundColor.startsWith("rgba") && activeTab === "Solid") {
+            // Buttons in the solid tab of color selector have no
+            // opacity, hence to match selected color correctly,
+            // we need to remove applied 0.6 opacity.
+            const values = backgroundColor.match(/[\d.]{1,5}/g) || [];
```

Comment:
It was better as a constant in `color.js`. Please move that back there even if it's now part of the web module.